### ISSUE TITLE
delete-build-on-push New Issue - 

### ISSUE THREAD

**Issue by krishna-vanvlack** [VanVlackApplication/JenkinsChanges#26]
The VanVlack application did a WebHook Push and the Jenkins Server Received it. 
@krishna-vanvlack pay attention to this. 

@colossus9, @nathos, and @mattcantstop this is just a merge Webhook.

**Comment by krishna-vanvlack** [comment]
The VanVlack application did a WebHook Push and the Jenkins Server Received it. 
@krishna-vanvlack pay attention to this. 

@colossus9, @nathos, and @mattcantstop this is just a merge Webhook. 

Build 'delete-build-on-push' is failing!

Last 50 lines of build output:

```
Started by GitHub push by krishna-vanvlack
[EnvInject] - Loading node environment variables.
Building in workspace /var/lib/jenkins/workspace/delete-build-on-push
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.deleted_build.url https://github.com/VanVlackApplication/JenkinsDelete # timeout=10
Fetching upstream changes from https://github.com/VanVlackApplication/JenkinsDelete
 > git --version # timeout=10
using GIT_ASKPASS to set credentials me
 > git fetch --tags --progress https://github.com/VanVlackApplication/JenkinsDelete +refs/heads/*:refs/remotes/deleted_build/*
 > git rev-parse deleted_build/master^{commit} # timeout=10
Checking out Revision 7df5fde3cc7feb6cb8078cfc56e0a6bfa2eda290 (deleted_build/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 7df5fde3cc7feb6cb8078cfc56e0a6bfa2eda290
Commit message: "Delete updated"
 > git rev-list --no-walk b98a6361df87d159c27bdf7a7dd55c05a98a518d # timeout=10
[Set GitHub commit status (universal)] PENDING on repos [GHRepository@1543dca[description=this is the repo that is deleted,homepage=<null>,name=JenkinsDelete,license=<null>,fork=false,size=43,milestones={},language=<null>,commits={},source=<null>,parent=<null>,responseHeaderFields={null=[HTTP/1.1 200 OK], Access-Control-Allow-Origin=[*], Access-Control-Expose-Headers=[ETag, Link, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval], Cache-Control=[private, max-age=60, s-maxage=60], Content-Encoding=[gzip], Content-Security-Policy=[default-src 'none'], Content-Type=[application/octet-stream], Date=[Mon, 08 Jan 2018 03:59:59 GMT], ETag=["dd2b8995cc29e1830e3a842bae971baa"], Last-Modified=[Mon, 08 Jan 2018 03:34:27 GMT], OkHttp-Received-Millis=[1515383999374], OkHttp-Response-Source=[CACHE 200], OkHttp-Selected-Protocol=[http/1.1], OkHttp-Sent-Millis=[1515383999242], Server=[GitHub.com], Status=[304 Not Modified], Strict-Transport-Security=[max-age=31536000; includeSubdomains; preload], Transfer-Encoding=[chunked], Vary=[Accept, Authorization, Cookie, X-GitHub-OTP], X-Accepted-OAuth-Scopes=[repo], X-Content-Type-Options=[nosniff], X-Frame-Options=[deny], X-GitHub-Media-Type=[github.v3; format=json], X-GitHub-Request-Id=[D14C:6560:142309A:297AEE0:5A52ECBF], X-OAuth-Scopes=[admin:gpg_key, admin:org, admin:org_hook, admin:public_key, admin:repo_hook, delete_repo, gist, notifications, repo, user], X-RateLimit-Limit=[5000], X-RateLimit-Remaining=[4920], X-RateLimit-Reset=[1515387541], X-Runtime-rack=[0.052735], X-XSS-Protection=[1; mode=block]},url=https://api.github.com/repos/VanVlackApplication/JenkinsDelete,id=116318287]] (sha:7df5fde) with context:organization
Setting commit status on GitHub for https://github.com/VanVlackApplication/JenkinsDelete/commit/7df5fde3cc7feb6cb8078cfc56e0a6bfa2eda290
[delete-build-on-push] $ /bin/sh -xe /tmp/hudson7219307488410664563.sh
+ ghi
/tmp/hudson7219307488410664563.sh: line 2: ghi: command not found
Build step 'Execute shell' marked build as failure

```

Changes since last successful build:
- [noreply] 183bc3605f32fe0460b5e60ae2c519cbeccbba31 - testing a hook

- [noreply] 7df5fde3cc7feb6cb8078cfc56e0a6bfa2eda290 - Delete updated



[View full output](http://18.217.248.72:8080/job/delete-build-on-push/7/)

**Comment by krishna-vanvlack** [comment]
The VanVlack application did a WebHook Push and the Jenkins Server Received it. 
@krishna-vanvlack pay attention to this. 

@colossus9, @nathos, and @mattcantstop this is just a merge Webhook. 

Build 'delete-build-on-push' is failing!

Last 50 lines of build output:

```
Started by GitHub push by krishna-vanvlack
[EnvInject] - Loading node environment variables.
Building in workspace /var/lib/jenkins/workspace/delete-build-on-push
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.deleted_build.url https://github.com/VanVlackApplication/JenkinsDelete # timeout=10
Fetching upstream changes from https://github.com/VanVlackApplication/JenkinsDelete
 > git --version # timeout=10
using GIT_ASKPASS to set credentials me
 > git fetch --tags --progress https://github.com/VanVlackApplication/JenkinsDelete +refs/heads/*:refs/remotes/deleted_build/*
 > git rev-parse deleted_build/master^{commit} # timeout=10
Checking out Revision 576bbf10001ee805262997b1a5293095bf2ccdeb (deleted_build/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 576bbf10001ee805262997b1a5293095bf2ccdeb
Commit message: "Merge pull request #31 from VanVlackApplication/deleted"
 > git rev-list --no-walk 7df5fde3cc7feb6cb8078cfc56e0a6bfa2eda290 # timeout=10
[Set GitHub commit status (universal)] PENDING on repos [GHRepository@137c92df[description=this is the repo that is deleted,homepage=<null>,name=JenkinsDelete,license=<null>,fork=false,size=43,milestones={},language=<null>,commits={},source=<null>,parent=<null>,responseHeaderFields={null=[HTTP/1.1 200 OK], Access-Control-Allow-Origin=[*], Access-Control-Expose-Headers=[ETag, Link, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval], Cache-Control=[private, max-age=60, s-maxage=60], Content-Encoding=[gzip], Content-Security-Policy=[default-src 'none'], Content-Type=[application/json; charset=utf-8], Date=[Mon, 08 Jan 2018 04:06:12 GMT], ETag=[W/"5416177077aa7a9747e29b5e6e8b8705"], Last-Modified=[Mon, 08 Jan 2018 03:34:27 GMT], OkHttp-Received-Millis=[1515384372450], OkHttp-Response-Source=[CACHE 200], OkHttp-Selected-Protocol=[http/1.1], OkHttp-Sent-Millis=[1515384372340], Server=[GitHub.com], Status=[200 OK], Strict-Transport-Security=[max-age=31536000; includeSubdomains; preload], Transfer-Encoding=[chunked], Vary=[Accept, Authorization, Cookie, X-GitHub-OTP], X-Accepted-OAuth-Scopes=[repo], X-Content-Type-Options=[nosniff], X-Frame-Options=[deny], X-GitHub-Media-Type=[github.v3; format=json], X-GitHub-Request-Id=[C82A:6560:1428517:2986277:5A52EE34], X-OAuth-Scopes=[admin:gpg_key, admin:org, admin:org_hook, admin:public_key, admin:repo_hook, delete_repo, gist, notifications, repo, user], X-RateLimit-Limit=[5000], X-RateLimit-Remaining=[4611], X-RateLimit-Reset=[1515387541], X-Runtime-rack=[0.047946], X-XSS-Protection=[1; mode=block]},url=https://api.github.com/repos/VanVlackApplication/JenkinsDelete,id=116318287]] (sha:576bbf1) with context:organization
Setting commit status on GitHub for https://github.com/VanVlackApplication/JenkinsDelete/commit/576bbf10001ee805262997b1a5293095bf2ccdeb
[delete-build-on-push] $ /bin/sh -xe /tmp/hudson6087216407887099206.sh
+ ghi
/tmp/hudson6087216407887099206.sh: line 2: ghi: command not found
Build step 'Execute shell' marked build as failure

```

Changes since last successful build:
- [noreply] 183bc3605f32fe0460b5e60ae2c519cbeccbba31 - testing a hook

- [noreply] 7df5fde3cc7feb6cb8078cfc56e0a6bfa2eda290 - Delete updated

- [noreply] 8ce5503394346ea50942c3c6928b6ee2ab88179c - delete



[View full output](http://18.217.248.72:8080/job/delete-build-on-push/8/)

**Comment by krishna-vanvlack** [comment]
The VanVlack application did a WebHook Push and the Jenkins Server Received it. 
@krishna-vanvlack pay attention to this. 

@colossus9, @nathos, and @mattcantstop this is just a merge Webhook. 

Build 'delete-build-on-push' is failing!

Last 50 lines of build output:

```
Started by user admin
[EnvInject] - Loading node environment variables.
Building in workspace /var/lib/jenkins/workspace/delete-build-on-push
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.deleted_build.url https://github.com/VanVlackApplication/JenkinsDelete # timeout=10
Fetching upstream changes from https://github.com/VanVlackApplication/JenkinsDelete
 > git --version # timeout=10
using GIT_ASKPASS to set credentials me
 > git fetch --tags --progress https://github.com/VanVlackApplication/JenkinsDelete +refs/heads/*:refs/remotes/deleted_build/*
 > git rev-parse deleted_build/master^{commit} # timeout=10
Checking out Revision 576bbf10001ee805262997b1a5293095bf2ccdeb (deleted_build/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 576bbf10001ee805262997b1a5293095bf2ccdeb
Commit message: "Merge pull request #31 from VanVlackApplication/deleted"
 > git rev-list --no-walk 576bbf10001ee805262997b1a5293095bf2ccdeb # timeout=10
[Set GitHub commit status (universal)] PENDING on repos [GHRepository@41a2be0a[description=this is the repo that is deleted,homepage=<null>,name=JenkinsDelete,license=<null>,fork=false,size=49,milestones={},language=<null>,commits={},source=<null>,parent=<null>,responseHeaderFields={null=[HTTP/1.1 200 OK], Access-Control-Allow-Origin=[*], Access-Control-Expose-Headers=[ETag, Link, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval], Cache-Control=[private, max-age=60, s-maxage=60], Content-Encoding=[gzip], Content-Security-Policy=[default-src 'none'], Content-Type=[application/json; charset=utf-8], Date=[Mon, 08 Jan 2018 04:53:44 GMT], ETag=[W/"847200de781c956eac669fec6a5f29d1"], Last-Modified=[Mon, 08 Jan 2018 04:14:55 GMT], OkHttp-Received-Millis=[1515387224586], OkHttp-Response-Source=[CONDITIONAL_CACHE 200], OkHttp-Selected-Protocol=[http/1.1], OkHttp-Sent-Millis=[1515387224464], Server=[GitHub.com], Status=[200 OK], Strict-Transport-Security=[max-age=31536000; includeSubdomains; preload], Transfer-Encoding=[chunked], Vary=[Accept, Authorization, Cookie, X-GitHub-OTP], X-Accepted-OAuth-Scopes=[repo], X-Content-Type-Options=[nosniff], X-Frame-Options=[deny], X-GitHub-Media-Type=[github.v3; format=json], X-GitHub-Request-Id=[D2AA:6562:1FE610D:3BDF713:5A52F958], X-OAuth-Scopes=[admin:gpg_key, admin:org, admin:org_hook, admin:public_key, admin:repo_hook, delete_repo, gist, notifications, repo, user], X-RateLimit-Limit=[5000], X-RateLimit-Remaining=[578], X-RateLimit-Reset=[1515387541], X-Runtime-rack=[0.060492], X-XSS-Protection=[1; mode=block]},url=https://api.github.com/repos/VanVlackApplication/JenkinsDelete,id=116318287]] (sha:576bbf1) with context:organization
Setting commit status on GitHub for https://github.com/VanVlackApplication/JenkinsDelete/commit/576bbf10001ee805262997b1a5293095bf2ccdeb
[delete-build-on-push] $ /bin/sh -xe /tmp/hudson7119832862888718770.sh
+ ghi
/tmp/hudson7119832862888718770.sh: line 2: ghi: command not found
Build step 'Execute shell' marked build as failure

```

Changes since last successful build:
- [noreply] 183bc3605f32fe0460b5e60ae2c519cbeccbba31 - testing a hook

- [noreply] 7df5fde3cc7feb6cb8078cfc56e0a6bfa2eda290 - Delete updated

- [noreply] 8ce5503394346ea50942c3c6928b6ee2ab88179c - delete



[View full output](http://18.217.248.72:8080/job/delete-build-on-push/9/)

**Comment by krishna-vanvlack** [comment]
The VanVlack application did a WebHook Push and the Jenkins Server Received it. 
@krishna-vanvlack pay attention to this. 

@colossus9, @nathos, and @mattcantstop this is just a merge Webhook. 

Build 'delete-build-on-push' is failing!

Last 50 lines of build output:

```
Started by user admin
[EnvInject] - Loading node environment variables.
Building in workspace /var/lib/jenkins/workspace/delete-build-on-push
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.deleted_build.url https://github.com/VanVlackApplication/JenkinsDelete # timeout=10
Fetching upstream changes from https://github.com/VanVlackApplication/JenkinsDelete
 > git --version # timeout=10
using GIT_ASKPASS to set credentials me
 > git fetch --tags --progress https://github.com/VanVlackApplication/JenkinsDelete +refs/heads/*:refs/remotes/deleted_build/*
 > git rev-parse deleted_build/master^{commit} # timeout=10
Checking out Revision 576bbf10001ee805262997b1a5293095bf2ccdeb (deleted_build/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 576bbf10001ee805262997b1a5293095bf2ccdeb
Commit message: "Merge pull request #31 from VanVlackApplication/deleted"
 > git rev-list --no-walk 576bbf10001ee805262997b1a5293095bf2ccdeb # timeout=10
[Set GitHub commit status (universal)] PENDING on repos [GHRepository@16b0dcf7[description=this is the repo that is deleted,homepage=<null>,name=JenkinsDelete,license=<null>,fork=false,size=49,milestones={},language=<null>,commits={},source=<null>,parent=<null>,responseHeaderFields={null=[HTTP/1.1 200 OK], Access-Control-Allow-Origin=[*], Access-Control-Expose-Headers=[ETag, Link, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval], Cache-Control=[private, max-age=60, s-maxage=60], Content-Encoding=[gzip], Content-Security-Policy=[default-src 'none'], Content-Type=[application/json; charset=utf-8], Date=[Mon, 08 Jan 2018 04:53:44 GMT], ETag=[W/"847200de781c956eac669fec6a5f29d1"], Last-Modified=[Mon, 08 Jan 2018 04:14:55 GMT], OkHttp-Received-Millis=[1515387224586], OkHttp-Response-Source=[CACHE 200], OkHttp-Selected-Protocol=[http/1.1], OkHttp-Sent-Millis=[1515387224464], Server=[GitHub.com], Status=[200 OK], Strict-Transport-Security=[max-age=31536000; includeSubdomains; preload], Transfer-Encoding=[chunked], Vary=[Accept, Authorization, Cookie, X-GitHub-OTP], X-Accepted-OAuth-Scopes=[repo], X-Content-Type-Options=[nosniff], X-Frame-Options=[deny], X-GitHub-Media-Type=[github.v3; format=json], X-GitHub-Request-Id=[D2AA:6562:1FE610D:3BDF713:5A52F958], X-OAuth-Scopes=[admin:gpg_key, admin:org, admin:org_hook, admin:public_key, admin:repo_hook, delete_repo, gist, notifications, repo, user], X-RateLimit-Limit=[5000], X-RateLimit-Remaining=[578], X-RateLimit-Reset=[1515387541], X-Runtime-rack=[0.060492], X-XSS-Protection=[1; mode=block]},url=https://api.github.com/repos/VanVlackApplication/JenkinsDelete,id=116318287]] (sha:576bbf1) with context:organization
Setting commit status on GitHub for https://github.com/VanVlackApplication/JenkinsDelete/commit/576bbf10001ee805262997b1a5293095bf2ccdeb
[delete-build-on-push] $ /bin/sh -xe /tmp/hudson227454718612728706.sh
+ ghi
/tmp/hudson227454718612728706.sh: line 2: ghi: command not found
Build step 'Execute shell' marked build as failure

```

Changes since last successful build:
- [noreply] 183bc3605f32fe0460b5e60ae2c519cbeccbba31 - testing a hook

- [noreply] 7df5fde3cc7feb6cb8078cfc56e0a6bfa2eda290 - Delete updated

- [noreply] 8ce5503394346ea50942c3c6928b6ee2ab88179c - delete



[View full output](http://18.217.248.72:8080/job/delete-build-on-push/10/)
